_posts/en/2019-03-19-bech32-sending-support.md

```diff
@@ -141,5 +141,11 @@ on bech32 sending support, from March 19th to August 28th, 2019.
 
 {% include specials/bech32/19-real-fees.md %}
 
+## A decline in bech32 adoption?
```

Comment:
I don't think there is much risk in this, but for readers who skim "headlines" might get the wrong impression if they don't read the details. An alternative to consider would be something along the lines of "Misleading statistics..." or "Always healthy to dive into the details of bech32 stats."

Reply:
I agree that this should be changed, but I think something entirely neutral like "bech32 adoption rate" or similar is best.